### ISSUE TITLE
AttributeError: 'SingleRelatedObjectDescriptor' object has no attribute 'field'

### ISSUE THREAD

**Issue by ppo** [magopian/django-data-exports#1]
After clicking on "Save" while creating a new export on model E (see below).
## Traceback

File "virtualenv/lib/python2.6/site-packages/data_exports/forms.py", line 41, in add_fields
`choices = get_choices(model)`
`>>> model = <E>`

File "virtualenv/lib/python2.6/site-packages/data_exports/forms.py", line 58, in get_choices
`choices += get_choices(related_model, prefixes=new_prefixes)`
`>>> model = <D>`

File "virtualenv/lib/python2.6/site-packages/data_exports/forms.py", line 58, in get_choices
`choices += get_choices(related_model, prefixes=new_prefixes)`
`>>> model = <C>`

File "virtualenv/lib/python2.6/site-packages/data_exports/forms.py", line 58, in get_choices
`choices += get_choices(related_model, prefixes=new_prefixes)`
`>>> model = <A>`

File "virtualenv/lib/python2.6/site-packages/data_exports/forms.py", line 54, in get_choices
`related_model = getattr(model, f).field.rel.to`
`>>> model = <A>`
## Classes

``` python
class A(models.Model):
  # some fields

class B(A):
  # some fields

class C(A):
  b = models.ForeignKey(B)
  # some fields

class D(A):
  b = models.ForeignKey(B)
  c = models.ForeignKey(C)
  # some fields

class E(models.Model):
  d = models.ForeignKey(D)
  b = models.ForeignKey(B)
  # some fields
```
## Environment

Python 2.6.5
Django==1.3.1
django-data-exports==0.2
django-inspect-model==0.4

**Comment by magopian** [comment]
Thanks for the report, i'll look into that asap!

**Comment by ppo** [comment]
With Django==1.4

**AttributeError: 'ExportAdmin' object has no attribute 'inline_instances'**

## Traceback

File "virtualenv/lib/python2.6/site-packages/django/contrib/admin/options.py", line 1063, in change_view
`for FormSet, inline in zip(self.get_formsets(request, obj), inline_instances):`
`>>> obj = <E>`
`>>> inline_instances = [<data_exports.admin.ColumnInline>]`

File "virtualenv/lib/python2.6/site-packages/data_exports/admin.py", line 34, in get_formsets
`for inline in self.inline_instances:`
`>>> self = <data_exports.admin.ExportAdmin>`

**Comment by pielgrzym** [comment]
@ppo - fixed this in the pull request if you're still interested.

**Comment by magopian** [comment]
fixed by @pielgrzym pull request #2

**Comment by ppo** [comment]
Thanks, I'll give it another try :-)
